app/main/views/suppliers.py

```diff
@@ -444,3 +443,28 @@ def create_your_account_complete():
         "suppliers/create_your_account_complete.html",
         email_address=email_address
     ), 200
+
+
+@main.route('/<framework_slug>/opportunities-overview', methods=['GET'])
+@login_required
+def opportunities_overview(framework_slug):
```

Comment:
I might be missing something but why does this need to take framework as a param?  I can see it makes sense from the point of view of where the link is on the dashboard, but I can't think of any other reason to key on framework?  i.e. if we just had a "View my opportunities" link from the dashboard, not in the frameworks list, and showed a list of all brief responses on whatever framework that would probably work?

As a supplier I just want to see all the things I applied for, I don't particularly care what framework it was on?